### PR TITLE
76664 create sap mock api endpoint

### PR DESCRIPTION
**Pull request by pravinvarade10801**
PBI#76664 - Created SAP mock service
PBI#75446 - Consumed the mock service in ERP Facade.
PBI#76752 - Deploy SAP mock service

### CODE REVIEW

File: src/UKHO.ERPFacade.Common/Configuration/ISapConfiguration.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿namespace UKHO.ERPFacade.Common.Configuration
+{
+    public interface ISapConfiguration
+    {
+        public string BaseAddress { get; set; }
+
+        public string SapServiceOperation { get; set; }
+
+        public string Username { get; set; }
```
Comment (harshal11869) [inline src/UKHO.ERPFacade.Common/Configuration/ISapConfiguration.cs:9]:
```suggestion
        public string UserName { get; set; }
```

File: src/UKHO.ERPFacade.Common/Configuration/SapConfiguration.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+namespace UKHO.ERPFacade.Common.Configuration
+{
+    [ExcludeFromCodeCoverage]
+    public class SapConfiguration : ISapConfiguration
+    {
+        public string BaseAddress { get; set; }
+
+        public string SapServiceOperation { get; set; }
+
+        public string Username { get; set; }
```
Comment (harshal11869) [inline src/UKHO.ERPFacade.Common/Configuration/SapConfiguration.cs:12]:
```suggestion
        public string UserName { get; set; }
```

File: src/UKHO.ERPFacade.Common/Configuration/ISapConfiguration.cs (hunk 1 above)
Comment (Vishal14583) [reply src/UKHO.ERPFacade.Common/Configuration/ISapConfiguration.cs:9]: This is added as per keyvault configuration for SAP

File: src/UKHO.ERPFacade.Common/Configuration/SapConfiguration.cs (hunk 2 above)
Comment (Vishal14583) [reply src/UKHO.ERPFacade.Common/Configuration/SapConfiguration.cs:12]: This is added as per keyvault configuration for SAP.

**Comment by pravinvarade10801** [comment]
> 

This files are related to CI/CD. @amogh14893 @kiran1501497 please look into this.

**Review by mattwin1 — APPROVED** [review]
LGTM

**Review by jrippington — APPROVED** [review]
Looks good, thanks!

**Review by harshal11869 — APPROVED** [review]
Looks Good.

**Review by JiviteshT — APPROVED** [review]
Approved to progress merge

**Pull request merged**